WinUIGallery/Samples/SamplePages/SampleWindow3.xaml.cs

```diff
@@ -41,7 +42,13 @@ private void MaximizeBtn_Click(object sender, RoutedEventArgs e)
     private void MinimizeBtn_Click(object sender, RoutedEventArgs e)
     {
         presenter.Minimize();
-    }
+    }
+
+    private void MaximizeRestoreBtn_Click(object sender, RoutedEventArgs e)
+    {
+        presenter.Maximize();
+        Task.Delay(3000).ContinueWith(t => presenter.Restore());

```

Comment:
I think using a ToggleButton would be better than making the user wait 3 seconds.